include/librealsense2/rsutil.h

```diff
@@ -4,6 +4,10 @@
 #ifndef LIBREALSENSE_RSUTIL2_H
 #define LIBREALSENSE_RSUTIL2_H
 
+#ifdef __cplusplus
```

Comment:
These should be after all the `#include` statements

Reply:
Done